src/components/calcite-flow-item/calcite-flow-item.tsx

```diff
@@ -179,14 +187,20 @@ export class CalciteFlowItem {
   }
 
   render() {
+    const { el, showBackButton, heading } = this;
+
+    const rtl = getElementDir(el) === "rtl";
+    this.rtl = rtl;
```

Comment:
I think computing RTL during render calls is fine, no need to store the value. If you want to avoid recomputing it a second time, I'd suggest adding a parameter to `renderBackButton` instead.